vars/Fedora.yml

```diff
@@ -6,8 +6,11 @@ format: rpm
 # should be installed.  Note that we force the distribution name here,
 # because what Nessus uses ("fc") does not match what Ansible uses
 # ("Fedora").
+#
+# As of Nessus 10.9.4, the RPMs are labeled as "el" (Enterprise Linux)
+# not "fc" (Fedora Core).
 package_file_regex: >-
-  Nessus-{{ nessus_version }}-fc[0-9]*\.{{ ansible_architecture }}\.{{ format }}
+  Nessus-{{ nessus_version }}-(el|fc)[0-9]+\.{{ ansible_architecture }}\.{{ format }}
```

Comment:
Did older versions still include a number after the `fc`? Otherwise this could break using an older Nessus installer package.